amivapi/templates/events_accept_calendar_invite.ics

```diff
@@ -5,7 +5,6 @@ X-MS-OLK-FORCEINSPECTOROPEN:TRUE
 METHOD:PUBLISH
 BEGIN:VEVENT
 UID:{{ event_id }}@amiv.ethz.ch
-ORGANIZER:MAILTO:{{ reply_to_email }}
```

Comment:
If a client does not support the organizer field, wouldn't it just be ignored?

Reply:
should this now be done this way?

Reply:
apperently no client supports it, and outlook has active problems with it. (some formating breaks for the entire email). therefore it's better to leave it away.